edx_proctoring/api.py

```diff
@@ -115,58 +115,61 @@ def create_exam(course_id, content_id, exam_name, time_limit_mins, due_date=None
     return proctored_exam.id
 
 
-def create_exam_review_policy(exam_id, set_by_user_id, review_policy):
+def create_exam_review_policy(exam_id, set_by_user_id, review_policy, rules):
     """
     Creates a new exam_review_policy entity, if the review_policy
     for exam_id does not already exist. If it exists, then raise exception.
 
     Returns: id (PK)
     """
-
     exam_review_policy = ProctoredExamReviewPolicy.get_review_policy_for_exam(exam_id)
     if exam_review_policy is not None:
         raise ProctoredExamReviewPolicyAlreadyExists
 
     exam_review_policy = ProctoredExamReviewPolicy.objects.create(
         proctored_exam_id=exam_id,
         set_by_user_id=set_by_user_id,
-        review_policy=review_policy
+        review_policy=review_policy,
+        rules=rules
     )
 
     log_msg = (
         u'Created ProctoredExamReviewPolicy ({review_policy}) with parameters: exam_id={exam_id}, '
-        u'set_by_user_id={set_by_user_id}'.format(
+        u'set_by_user_id={set_by_user_id}, rules={rules}'.format(
             exam_id=exam_id,
             review_policy=review_policy,
-            set_by_user_id=set_by_user_id
+            set_by_user_id=set_by_user_id,
+            rules=rules
         )
     )
     log.info(log_msg)
 
     return exam_review_policy.id
 
 
-def update_review_policy(exam_id, set_by_user_id, review_policy):
+def update_review_policy(exam_id, set_by_user_id, review_policy, rules):
     """
     Given a exam id, update/remove the existing record, otherwise raise exception if not found.
     Returns: review_policy_id
     """
-
     log_msg = (
         u'Updating exam review policy with exam_id {exam_id}'
         u'set_by_user_id={set_by_user_id}, review_policy={review_policy}'
+        u'rules={rules}'
         .format(
             exam_id=exam_id, set_by_user_id=set_by_user_id, review_policy=review_policy,
+            rules=rules
         )
     )
     log.info(log_msg)
     exam_review_policy = ProctoredExamReviewPolicy.get_review_policy_for_exam(exam_id)
     if exam_review_policy is None:
         raise ProctoredExamReviewPolicyNotFoundException
 
-    if review_policy:
+    if review_policy or rules:
```

Comment:
TODO: This has the potential to save None as the review policy, which the ProctoredExamReviewPolicy model does not support (i.e. `review_policy = models.TextField()`). I would have to do allow null values. Is this the right approach?

Reply:
How about ``review_policy = models.TextField(default='')``? I don't think that would require a migration.

Reply:
It did create a migration, actually. Does that change your suggestion?

Reply:
No, I guess that's ok.